main.html

```diff
@@ -164,31 +164,49 @@ <h1 class="primary-text">About <br>Myself</h1>
-                <li class="CSS"><img src="images/mobile_menu_images/CSS.jpg" class="CSS-icon" alt="Cascading Style Sheets"><span class="CSS-text">CSS</span></li>
-            </ul>
+    <div class="skill-list">
+        <div class="frame-list">
+            <div class="frame-2">
+                <h1 class="language-list"><b>Languages</b></h1>
+                <img src="images/mobile_menu_images/Normal-Button/Tertiary/Disabled.jpg" class="arrow-down" alt="arrow-down">
+                <ul class="programming-languages">
+                    <li class="JS"><img src="images/mobile_menu_images/JS.jpg" class="JS-icon" alt="JavaScript"><span class="JS-text">JavaScript</span></li>
+                    <li class="HTML"><img src="images/mobile_menu_images/HTML.jpg" class="HTML-icon" alt="HyperText Mark-up Language"><span class="HTML-text">HTML</span></li>
+                    <li class="CSS"><img src="images/mobile_menu_images/CSS.jpg" class="CSS-icon" alt="Cascading Style Sheets"><span class="CSS-text">CSS</span></li>
+                </ul>
+            </div>
+        </div>
+        <div class="framework-layout">
+            <h1 class="framework-text"><b>Frameworks</b></h1>
+            <img src="images/mobile_menu_images/Normal-Button/Tertiary/Disabled.png" alt="right-arrow" class="arrow-right">
+        </div>
+        <div class="skill-list-layout">
+            <h1 class="skill-list-text"><b>Skills</b></h1>
+            <img src="images/mobile_menu_images/Normal-Button/Tertiary/Disabled.png" alt="right-arrow" class="arrow-right">
         </div>
     </div>
-    <div class="framework-layout">
-        <h1 class="framework-text"><b>Frameworks</b></h1>
-        <img src="images/mobile_menu_images/Normal-Button/Tertiary/Disabled.png" alt="right-arrow" class="arrow-right">
-    </div>
-    <div class="skill-list-layout">
-        <h1 class="skill-list-text"><b>Skills</b></h1>
-        <img src="images/mobile_menu_images/Normal-Button/Tertiary/Disabled.png" alt="right-arrow" class="arrow-right">
-    </div>
-</div>
 
-</section>   
+</section> 
+
+<footer class="contact-section">
+    <div class="background-shapes">
+        <div class="contact-list">
+            <h1>Contact me</h1>
+            <p>
+                If you have an application you are interested in developing, a feature that you need build, or a project that needs coding I'd love to help
+            </p>
+        </div>
+        <form action="https://formspree.io/f/xwkyyzoa" method="post">
+            <input type="text" name="name" maxlength="30" id="name" required placeholder="Enter your full name"/>
+            <input type="email" name="email" maxlength="30" id="email" required placeholder="name@domain.com" />
+            <textarea maxlength="500" placeholder="Write your message here" name="message"></textarea>
+            <button type="submit">Get in Touch</button>
```

Comment:
- [ ] Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) should be styled as shown in the Figma template.